js/api/Geofencing.js

```diff
@@ -129,6 +130,12 @@ export function setupGeofences(callback) {
       supressNotificationsTimestamp = Date.now() + 5000;
       BackgroundGeolocation.addGeofences(triggerGeofences, () => {
         console.log('Successfully added geofences.');
+
+        // Restarts the service on Android devices.
+        // This redundancy is a patch to force geofences to be rechecked in some devices.
+        if (Platform.OS === 'android') {
```

Comment:
Mine.

Although it may be related to the bug you mentioned on the comment just below.
I will remove this on the next update if my geofences resume working as intended.